README.md

```diff
@@ -571,6 +572,21 @@ To have a more interesting port, the following features would need to be impleme
 * A disk to store the `init.bin`/romdisk, can be read-only, so can be stored on the ROM
 * A read-write disk to store data, can be a floppy disk driver using ZealFS filesystem
 
+## Agon Light
+
+A port to the eZ80 powered Agon Light, written and maintained by [Shawn Sijnstra](https://github.com/sijnstra/Zeal-8-bit-OS). Feel free to use that fork for Agon specific bugs/requests. This uses the non-MMU kernel, and implements as many of the features that the Zeal implementation supports.
```

Comment:
Did you mean  `the Zeal 8-bit Computer implementation supports` or `the Zeal 8-bit OS implementation supports`?

Reply:
Updated and clarified (I hope!)